utils/results/common.go

```diff
@@ -0,0 +1,612 @@
+package results
+
+import (
+	"fmt"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/jfrog/jfrog-cli-core/v2/utils/coreutils"
+	"github.com/jfrog/jfrog-cli-security/utils/formats"
+	"github.com/jfrog/jfrog-cli-security/utils/formats/sarifutils"
+	"github.com/jfrog/jfrog-cli-security/utils/jasutils"
+	"github.com/jfrog/jfrog-cli-security/utils/severityutils"
+	"github.com/jfrog/jfrog-cli-security/utils/techutils"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+	"github.com/jfrog/jfrog-client-go/xray/services"
+	"github.com/owenrumney/go-sarif/v2/sarif"
+	"golang.org/x/exp/slices"
+)
+
+const (
+	customLicenseViolationId   = "custom_license_violation"
+	RootIndex                  = 0
+	DirectDependencyIndex      = 1
+	DirectDependencyPathLength = 2
+	nodeModules                = "node_modules"
```

Comment:
should nodeModules be here? isn't that too npm specific?

Reply:
Ok - I'd prefer to take it from a more generic utils library - but not critical to me